src/client/javascripts/App.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    View,
+    Platform
+} from 'react-native';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        titleText: 'Yoda',
+        os: 'OS',
+      };
+
+    //For specific platform
+    if (Platform.OS === 'ios') {
+      this.state.os = 'IOS';
+    } else if (Platform.OS === 'android') {
+      this.state.os = 'Android API :' + Platform.Version;
+    }
+
+    console.log(Platform.OS);
```

Comment:
Where does the console log go to? Is this only for debugging, or does the production version also produce these logs? If this is for debugging, it needs to be more descriptive:

consol.log('Starting the app for platform: ' + this.state.os)

Also, note that I'm using this.state.os which is the newly populated state rather than making another call to Platform param. This way we can be sure that the if-block above has properly been written while debugging.